src/lib/transition_handler/processor.ml

```diff
@@ -67,14 +62,12 @@ module Make (Inputs : Inputs.With_unprocessed_transition_cache.S) :
     let catchup_scheduler =
       Catchup_scheduler.create ~logger ~frontier ~time_controller
         ~catchup_job_writer ~catchup_breadcrumbs_writer
-        ~clean_up_signal:clean_up_catchup_scheduler
     in
     (* add a breadcrumb and perform post processing *)
     let add_and_finalize ~only_if_present cached_breadcrumb =
       let open Deferred.Or_error.Let_syntax in
       let%bind breadcrumb =
-        Deferred.Or_error.return
-          (Cached.invalidate_with_success cached_breadcrumb)
+        Deferred.return (Cached.invalidate cached_breadcrumb)
```

Comment:
I don't see this function in the `Cached` interface. Shouldn't you use either `invalidate_with_success` or `invalidate_with_failure`?